.github/workflows/test.yml

```diff
@@ -3,6 +3,9 @@ on:
   pull_request:
     branches:
       - master
+  schedule:
+    - cron: 0 2 * * *
```

Comment:
please use a non exact our.
like 2:07, or 2:03 or 2:22

times like 2:00 are prone to errors as many frequent tasks are scheduled for these times.

Reply:
@tsnoam yeah, that was a copy paste error